search-provider/eks-discovery-feed-provider.c

```diff
@@ -807,7 +807,7 @@ handle_get_recent_news (EksDiscoveryFeedDatabaseContentProvider *skeleton,
                                                      "tags-match-any", tags_match_any,
                                                      "sort", EKNC_QUERY_OBJECT_SORT_DATE,
                                                      "order", EKNC_QUERY_OBJECT_ORDER_DESCENDING,
-                                                     "limit", 5,
+                                                     "limit", 1,
```

Comment:
Yeah fair point, I'll remove the iteration now. I still think that we should return an array type, just in case we want to return more in future.